pkg/probe/ha-checksum.go

```diff
@@ -0,0 +1,42 @@
+package probe
+
+import (
+	"log"
+
+	"github.com/bluecmd/fortigate_exporter/pkg/http"
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+type HAChecksumResults struct {
+	IsManageMaster int    `json:"is_manage_master"`
+	IsRootMaster   int    `json:"is_root_master"`
+	SerialNo       string `json:"serial_no"`
+}
+
+type HAChecksum struct {
+	Results []HAChecksumResults `json:"results"`
+}
+
+func probeSystemHAChecksum(c http.FortiHTTP, meta *TargetMetadata) ([]prometheus.Metric, bool) {
+	var (
+		IsMaster = prometheus.NewDesc(
+			"fortigate_ha_role",
```

Comment:
I think the metric name `fortigate_ha_member_has_role` is more aligned with the other HA metrics, thoughts?

Reply:
Totally agree